content/competitions/hackathon-2024.html

```diff
@@ -0,0 +1,68 @@
+---
+title: "Hackathon 2024"
+date: 2024-07-26T22:14:00+10:00
+draft: false
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+
+        <div class="block">
+            <a href="https://fb.me/e/7qY5282Ut">
```

Comment:
link doesn't seem to work on my phone? possibly a me issue, feel free to disregard. ( this does: https://facebook.com/events/s/uqcs-hackathon-2024/497939952616194/ )

Reply:
Seems to work on desktop and my phone, and follows same pattern as last year. I'll keep as is for now.